deploy/treasureMainnet/TreasureMarketplace.deploy.ts

```diff
@@ -0,0 +1,114 @@
+import * as hre from 'hardhat';
+import { HttpNetworkUserConfig } from 'hardhat/types';
+import { Deployer } from '@matterlabs/hardhat-zksync-deploy';
+import { Provider, Wallet } from 'zksync-ethers';
+
+const func = async () => {
+    const wallet = getWallet();
+
+    const deployer = new Deployer(hre, wallet);
+
+    // Constants for this deploy script.
+    const fee = 500n; // 5%
+    const feeWithCollectionOwner = 250n; // 2.5%
+    const feeReceipient = deployer.zkWallet.address;
```

Comment:
Let's leave a `// TODO: change to Treasure Mainnet marketplace multisig when deployed`.